spec/presenters/tree_builder_ae_class_spec.rb

```diff
@@ -22,4 +22,25 @@
       domains.should match_array %w(LUIGI MARIO)
     end
   end
+
+  context "#x_get_tree_roots" do
+    before do
+      root_tenant = EvmSpecHelper.create_root_tenant
+      user = FactoryGirl.create(:user)
+      User.stub(:current_user).and_return(user)
+      user.stub(:current_tenant).and_return(root_tenant)
+      tenant2 = FactoryGirl.create(:tenant, :parent => root_tenant)
+      FactoryGirl.create(:miq_ae_domain, :name => "test1", :parent => nil, :tenant => root_tenant)
+      FactoryGirl.create(:miq_ae_domain, :name => "test2", :parent => nil, :tenant => root_tenant)
+      FactoryGirl.create(:miq_ae_domain, :name => "test3", :parent => nil, :tenant => tenant2)
+      FactoryGirl.create(:miq_ae_domain, :name => "test4", :parent => nil, :tenant => tenant2)
```

Comment:
@h-kataria 
We don't have to specify the parent when building the miq_ae_domain, its only needed for miq_ae_namespace.